pdfannots/cli.py

```diff
@@ -17,6 +17,7 @@
     'print_filename',
     'sections',
     'use_page_labels',
+    'page_number_offset',
```

Comment:
nit: please keep this sorted, i.e. move to line 17

Reply:
Done.